guide/src/mocking_lros.md

```diff
@@ -157,6 +157,80 @@ response.
 {{#rustdoc_include ../samples/tests/mocking_lros_manual.rs:client-call}}
 ```
 
+## Simulating errors
+
+Errors can arise in an LRO from a few places.
+
+If your application uses automatic polling, the following cases are all
+equivalent: `until_done()` returns the error in the `Result`, regardless of
+where it originated.
+[Simulating an error starting an LRO](#simulating-an-error-starting-an-lro)
+will yield the simplest test.
+
+### Simulating an error starting an LRO
+
+The simplest way to simulate an error is to have the initial request fail with
+an error.
+
+```rust,ignore,noplayground
+{{#rustdoc_include ../samples/tests/mocking_lros_error.rs:expectation-initial}}
+```
```

Comment:
This is subject to the retry policy. You should say something about returning an error that is not retryable.

Reply:
The stubbed out clients do not have policies. I moved the blurb that says that up to the initial section of the error handling. (it was formerly tucked away in the last of the subsections).